packages/demo/demos/select.mjs

```diff
@@ -28,33 +57,18 @@ const demo = async () => {
     message: 'Select your favorite letter',
     choices: [
       new Separator('== Alphabet (choices cycle as you scroll through) =='),
-      { value: 'A' },
-      { value: 'B' },
-      { value: 'C' },
-      { value: 'D' },
-      { value: 'E' },
-      { value: 'F' },
-      { value: 'G' },
-      { value: 'H' },
-      { value: 'I' },
-      { value: 'J' },
-      { value: 'K' },
-      { value: 'L' },
-      { value: 'M' },
-      { value: 'N' },
-      { value: 'O', description: 'Letter O, not number 0' },
-      { value: 'P' },
-      { value: 'Q' },
-      { value: 'R' },
-      { value: 'S' },
-      { value: 'T' },
-      { value: 'U' },
-      { value: 'V' },
-      { value: 'W' },
-      { value: 'X' },
-      { value: 'Y' },
-      { value: 'Z' },
+      ...alphabet,
+    ],
+  });
+  console.log('Answer:', answer);
+
+  answer = await select({
+    message: 'Select your favorite letter (example without loop)',
+    choices: [
+      new Separator('== Alphabet (choices cycle as you scroll through) =='),
```

Comment:
There's a bug here when the first item is not selectable.

It'll allow the loop in that scenario.